contrib/CI/ubuntu.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash 
+
+set -e
+
+#https://grigorkh.medium.com/fix-tzdata-hangs-docker-image-build-cdb52cc3360d
+#Added because github runner would get stuck on interactive option when installing 
+#tzdata
+TZ="America/New_York"
```

Comment:
Denver

Reply:
@MigeljanImeri after you adjust this, feel free to merge it.